### PR TITLE
add question to profile resource

### PR DESCRIPTION
**Pull request by duckmoll**
#50 #52 

- AnswerQuestion for Profile Resource.
- Added Unit test to test question fields
- Reformat
- Below is the partial response from http://localhost:8000/api/v1/profile/:

screen shot 2018-12-08 at 3 34 03 pm

### CODE REVIEW

**Comment by duckmoll** [comment]
Sure thing.

**Comment by tangziyi001** [comment]
Try adding AnswerQuestion fetch logic and response append logic in dehydrate function. See UserRegisterResource as a reference

File: new_dawn_server/users/api/resources.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from django.db.models import signals
 from new_dawn_server.users.models import Account
 from new_dawn_server.users.models import Profile
+from new_dawn_server.questions.models import AnswerQuestion
```
Comment (tangziyi001) [inline new_dawn_server/users/api/resources.py]: alphabetical order

**Comment by tangziyi001** [comment]
FUCK YEAH!!!!!!!

**Comment by duckmoll** [comment]
> FUCK YEAH!!!!!!!

233333

**Pull request merged**